### DISCUSSION TITLE
Override crops for images

### DISCUSSION THREAD

**Discussion by nsivertsen** [area17/twill#998] [Ideas]
Hi again :)

Another question I'm facing is the following:

Twill makes it super easy to set a range of crops on an image for different applications. Sometimes, though, I'd actually like to have a different image for a specific crop, perhaps because this particular image doesn't work well in that ratio.

So, for example, if I have `thumbnail`, `'landscape` and `portrait` crops for an image, I might like to use a different image just for the `thumbnail` case.

Of course, I could just add more image fields for the different crops, but ideally, I'd only like to override a crop when necessary. Does that make sense?

Thanks, as always!

**Comment by ifox** [comment]
Hi @nsivertsen,

Perfectly makes sense! This is not possible right now apart from adding more image fields and using fallback logic when displaying in your frontend. It'd be a cool feature though! I'm not quite sure how this would look design and UX-wise yet but I'm going to keep this open as a potential enhancement for the future.
